core/app/models/spree/wished_product.rb

```diff
@@ -0,0 +1,16 @@
+module Spree
+  class WishedProduct < Spree::Base
+    belongs_to :variant
+    belongs_to :wishlist
```

Comment:
@MatthewKennedy we also need presence validation for variant and wishlist and uniqueness for the combination of the two